Makefile

```diff
@@ -98,16 +100,16 @@ dist: $(TARFILE)
 # pre-built dist/ (so it's not necessary) and ship packge-lock.json (so that
 # node_modules/ can be reconstructed if necessary)
 $(TARFILE): export NODE_ENV=production
-$(TARFILE): $(WEBPACK_TEST) $(SPEC)
+$(TARFILE): $(WEBPACK_TEST) $(SPEC) package.json
```

Comment:
What's that new dependency? Shouldn't that already be covered through the transitive WEBPACK_TEST → package.json dependency? If that is relevant, it should be its own separate commit.

Reply:
It's because of `--mtime ./package.json`, which sets the mtime for all the files. This way, if package.json changes, the tarballs are recreated (it is also mentioned in the description).

Reply:
Ack. So this is more about being explicit, than actually changing behaviour? Because it already works that way through the WEBPACK_TEST dependency:
```
❱❱❱ make dist
cockpit-starter-kit-1.tar.xz
(Nothing gets rebuilt)

❱❱❱ touch package.json; make dist
# if it exists already, npm install won't update it; force that so that we always get up-to-date packages
rm -f package-lock.json
# unset NODE_ENV, skips devDependencies otherwise
env -u NODE_ENV npm install
[...]
NODE_ENV= node_modules/.bin/webpack
[...]
```

Reply:
Yes, it seemed fair to do so (and I actually haven't checked whether there were other dependencies).